stm32-modules/heater-shaker/include/heater-shaker/heater_task.hpp

```diff
@@ -29,21 +62,43 @@ requires MessageQueue<QueueImpl<Message>, Message> class HeaterTask {
     static constexpr const uint32_t CONTROL_PERIOD_TICKS = 100;
     static constexpr double THERMISTOR_CIRCUIT_BIAS_RESISTANCE_KOHM = 44.2;
     static constexpr uint8_t ADC_BIT_DEPTH = 12;
+    static constexpr double HEATER_PAD_OVERTEMP_SAFETY_LIMIT_C = 100;
```

Comment:
Why aren't these typed as int32_t?

Reply:
Because of https://github.com/Opentrons/opentrons-modules/pull/163 mostly